wallet/src/Pos/Wallet/Web/Methods/History.hs

```diff
@@ -123,7 +133,11 @@ getHistoryLimited mCWalId mAccId mAddrId mSkip mLimit = do
             pure (cWalId', accIds')
         (Nothing, Just accId)   -> pure (aiWId accId, [accId])
     (unsortedThs, n) <- getHistory cWalId accIds mAddrId
-    let sortedTxh = sortByTime (Map.elems unsortedThs)
+
+    let !sortedTxh = sortByTime (Map.elems unsortedThs)
+    logDebug "getHistoryLimited: sorted transactions"
```

Comment:
I realize that this is not correct since I don't force full list evaluation, fixing..